packages/dm-core/src/components/UiRecipesSelector.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState, useContext } from 'react'
+
+import styled from 'styled-components'
+import { CircularProgress } from '@equinor/eds-core-react'
+import { useBlueprint } from '../hooks'
+import { UIPluginSelector } from './UiPluginSelector'
+import { UiRecipesSideBarSelector } from './UiRecipesSideBarSelector'
+import { UiPluginContext } from '../context/UiPluginContext'
+import { IUIPlugin } from '../types'
+
+const Wrapper = styled.div`
+  align-self: start;
+  justify-content: space-evenly;
+  width: 100%;
+`
+
+export function UIRecipesSelector(props: IUIPlugin): JSX.Element {
+  const { idReference, type } = props
+  const { initialUiRecipe, isLoading, error } = useBlueprint(type)
+  const { loading, getUiPlugin } = useContext(UiPluginContext)
```

Comment:
suggestion:
rename loading to isContextLoading
and isLoading to isBlueprintLoading